app/interactors/base_interactor.rb

```diff
@@ -0,0 +1,20 @@
+class BaseInteractor < ActiveInteraction::Base
+  def merging_errors(in_transaction: false)
```

Comment:
why is default transaction set to false?

Reply:
Great question! I guess i was being conservative with resources, but that's not a good reason at all. I'm down to switch it!